.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
```

Comment:
Files like this, and the .suo files probably shouldn't be committed when we merge into development, usually we're only editing the .ts, .html, .css, etc and not like these artifacts here